pages/linux/mono.md

```diff
@@ -0,0 +1,20 @@
+# mono
+
+> Mono, the open source development platform based on the .NET Framework.
+> More information: <https://www.mono-project.com/docs/>.
+
+- Compile:
+
+`csc {{source_file}}.cs`
+
+- Run:
+
+`mono {{source_file}}.exe`
```

Comment:
Perhaps this will read better / be more consistent with other pages?
```suggestion
`mono {{path/to/source_file.exe}}`
```

Reply:
```suggestion
`mono --debug {{path/to/source_file}}.exe`
```

Including `--debug` (and changing the description to something like "Run a .NET assembly in debug mode") may be even more helpful for devs as they work through bugs, general code, etc.

Reply:
Sounds good @Phrohdoh!